build/scripts/Build.fsx

```diff
@@ -18,6 +18,19 @@ open Paths
 open Products
 open Versions
 
+// use vswhere to set MSBuild location, if it exists. 
+// Allows FAKE MsBuildHelper.MsBuildRelease to work with Visual Studio 2019
+let vsWhere = Environment.ExpandEnvironmentVariables("%ProgramFiles(x86)%") </> @"Microsoft Visual Studio\Installer\vswhere.exe"
+if fileExists vsWhere then
+    let result = ExecProcessAndReturnMessages (fun p -> 
+                    p.FileName <- vsWhere
+                    p.Arguments <- @"-latest -requires Microsoft.Component.MSBuild -find MSBuild\**\Bin\MSBuild.exe"
```

Comment:
WDYT about searching for x64 msbuild with `**\amd64\msbuild.exe`?

Reply:
The commit has the example from the vswhere wiki for finding MSBuild:

https://github.com/microsoft/vswhere/wiki/Find-MSBuild

I don't think it matters which version of MSBuild is used - is there a reason you'd propose using 64bit MSBuild? I think the 32bit version is always available

Reply:
I figured you got it from that wiki page. No substantiated reason behind looking for x64 variant, a matter of preference I guess to use one first if it's available. Up to you here.